### PR TITLE
docs(config options): `improveprBodyTemplate` explanation

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Expand the `description` for the config option
- Drop the duplicate text from the docs for the config option
- Add `warnings` to list of available sections
- Convert to proper list

## Context

Drive-by PR, not closing any issue with this.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -2384,9 +2384,7 @@ e.g. if you wish to add an extra Warning to major updates:
 
 ## prBodyTemplate
 
-This setting controls which sections are rendered in the body of the pull request.
-
-The available sections are header, table, notes, changelogs, configDescription, controls, footer.
+The available sections are `header`, `table`, `warnings`, `notes`, `changelogs`, `configDescription`, `controls`, `footer`.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:2387]: The docs for `improveprBodyTemplate` [^docs] shows this default string:

> `"{{{header}}}{{{table}}}{{{warnings}}}{{{notes}}}{{{changelogs}}}{{{configDescription}}}{{{controls}}}{{{footer}}}"`

Does this mean that `warnings` should be added to the list of available sections?

[^docs]: https://docs.renovatebot.com/configuration-options/#prbodytemplate

File: lib/config/options/index.ts [hunk 2]
```diff
@@ -1807,7 +1807,7 @@ const options: RenovateOptions[] = [
   {
     name: 'prBodyTemplate',
```
Comment (HonkingGoose) [inline lib/config/options/index.ts:1808]: Not sure if this is a `docs:` or `chore:` type of change, please edit the PR title if needed. 😉

**Review by HonkingGoose — COMMENTED** [review]
Questions for maintainer. 😉

File: docs/usage/configuration-options.md (hunk 1 above)
Comment (rarkins) [reply docs/usage/configuration-options.md:2387]: Yes, I guess so

Comment (HonkingGoose) [reply docs/usage/configuration-options.md:2387]:
```suggestion
The available sections are:

- `header`
- `table`
- `warnings`
- `notes`
- `changelogs`
- `configDescription`
- `controls`
- `footer`
```

Let's convert this to a proper list while I'm at it anyways. 😄

**Review by rarkins — APPROVED** [review]

**Pull request merged**